requirements.txt

```diff
@@ -1 +1,151 @@
--r requirements/base.txt
+Django==1.8.10 \
+    --hash=sha256:471b41cb53d675138475b488c429424ed143e57ad755a2c8ab1206ac30490284 \
+    --hash=sha256:d2e5b11eb694984957378419f809b7205598326373d509d0262f9f84b17d1a23
+dj-database-url==0.3.0 \
+    --hash=sha256:f2e273ed34acbb560962d5cf12917936d8df02297df09bd3089b8546d4584138 \
+    --hash=sha256:ca01768fdecde134301f3170743226f60edff5c3935f12437378ebd911506353
+MySQL-python==1.2.3c1 \
```

Comment:
Moved `requirements/base.txt` to `requirements.txt`. Only real change to reqs in MySQL add, psycopg2 remove